telos-frontend/src/components/todo/DashboardTodo.jsx

```diff
@@ -1,5 +1,6 @@
+/* eslint-disable arrow-body-style */
+/* eslint-disable no-restricted-syntax */
```

Comment:
This is mainly out of curiosity, but why is eslint being disabled here?

Reply:
+1, I asked in a previous review but looks like the line I commented on was removed